### PR TITLE
feat(provider/kubernetes): list version annotation

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: reference/providers/kubernetes-v2/index.md [hunk 1]
```diff
@@ -120,6 +120,16 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
...
 
+## Strategy
+
+* `strategy.spinnaker.io/versioned`
+
+  When set to `true` or `false`, this overrides the resource's default
+  "version" behavior described in the [resource management
+  policies](#resource-management-policies). This can be used to force a
+  ConfigMap or Secret to be deployed without appending a new version when the
+  contents change, for example.
```
Comment (dorbin) [inline reference/providers/kubernetes-v2/index.md:131]: @lwander do you want to also add a sentence to the resource management policies section mentioning this?

**Review by dorbin — APPROVED** [review]
Thanks!

File: reference/providers/kubernetes-v2/index.md (hunk 1 above)
Comment (lwander) [reply reference/providers/kubernetes-v2/index.md:131]: Good idea!

Comment (lwander) [reply reference/providers/kubernetes-v2/index.md:131]: Done

Comment (dorbin) [reply reference/providers/kubernetes-v2/index.md:131]: :+1:

**Pull request merged**